### ISSUE TITLE
Suggestion: Let me point to a GitHub Repo URL in the extension's search repositories dialog

### ISSUE THREAD

**Issue by markw-t** [github/VisualStudio#792]
- GitHub Extension for Visual Studio  version: 2.1.1.4
- Visual Studio version: VS2017 RC

Wanted to open a GitHub repo using its URL. Clicked on the start page in VS and on the Checkout from: GitHub icon under the "Open" heading, expecting that would be where I could enter the URL.

In the Search Repositories dialog I pasted in the URL, and expected to be able to open, but it appears it doesn't support that. 

It would be great if I could just paste in my repo's URL and get to it from this open. I figured out how to do it from the repo's GitHub web page, which was great, but if I have the URL it'd be neat not to have to go to the web page.

**Comment by grokys** [comment]
Yep, I agree this would be nice! Would also be nice to be able to type just e.g. "github/VisualStudio" and get a clone.
